src/zope/ramcache/ram.py

```diff
@@ -15,6 +15,7 @@
 """
 __docformat__ = 'restructuredtext'
```

Comment:
Is this line actually still needed? It seems like cargo-cult to have it at the beginning of many ZF files.

Reply:
[zope.app.apidoc](https://github.com/zopefoundation/zope.app.apidoc/blob/master/src/zope/app/apidoc/utilities.py#L357) uses it. Up until the most recent release, the default was `structuredtext`, so this was quite necessary.